src/babel-plugin.js

```diff
@@ -0,0 +1,53 @@
+const { basename, extname } = require('path');
+
+module.exports = function hotReplaceAst({ types: t }) {
```

Comment:
function name seems not related to it's content

Reply:
Oops, copied from other my other code